### ISSUE TITLE
New bundle to get new dependencies like "tabularray"

### ISSUE THREAD

**Issue by openscript** [tectonic-typesetting/tectonic#871]
Thank you for the great tool! I've used it in numerous of documents.

I would love to use the `tabularray` (https://ctan.org/pkg/tabularray) package, but it seems like, that it isn't in the TexLive bundle from 2020. How can I update to a newer bundle or install `tabularray` manually?

**Comment by pkgw** [comment]
If you download the CTAN package for `tabularray` and stick its files into your document source directory ... hopefully it will work. For a lot of packages you just need the `.sty` file, and installing it alongside your source files will be OK.

But we are very overdue for a TeXLive system update, which would be the "right" way to fix this.

**Comment by openscript** [comment]
Thank you for updating to TexLive2021!

**Comment by Enter-tainer** [comment]
@openscript Is it fixed now? I'm still experiencing errors in v0.9.0

```
❯ tectonic --version
Tectonic 0.9.0


❯ tectonic report.tex
Running TeX ...
error: report.tex:51: ! LaTeX Error: File `tabularray.sty' not found.
error: halted on potentially-recoverable error as specified
```

**Comment by openscript** [comment]
It isn't included in https://data1.fullyjustified.net/tlextras-2021.3r1.tar, so not really, no.

What you can do is download the `tabularray.sty` and `ninecolors.sty` (dependency of tabularray which is also not in the bundle) and load it manually:

```
\usepackage{../lib/tabularray}
\usepackage{../lib/ninecolors}
```
